client/deis.py

```diff
@@ -1302,6 +1302,19 @@ def config_unset(self, args):
         else:
             raise ResponseError(response)
 
+    def _read_config_from_path(self, path):
+        env_dict = {}
+        with open(path, 'r') as f:
+            for line in f.readlines():
```

Comment:
Iterating directly on the [file object](https://docs.python.org/2/library/stdtypes.html#file.next) works too:
```python
for line in f:
```